v2/mfa/backend/second-factor.mdx

```diff
@@ -788,6 +1058,49 @@ Passwordless.init({
 TODO
 </TabItem>
 <TabItem value="python">
-TODO
+
+```python
+from supertokens_python.recipe.passwordless.interfaces import APIInterface, APIOptions
+from typing import Union, Dict, Any
+from supertokens_python.recipe.session.asyncio import get_session
+
+
+def override_passwordless_apis(original_implementation: APIInterface):
+    original_create_code_post = original_implementation.create_code_post
+
+    async def create_code_post(
+        email: Union[str, None],
+        phone_number: Union[str, None],
+        api_options: APIOptions,
+        user_context: Dict[str, Any],
+    ):
+        # This API is called to send an OTP
+        # We want to make sure that the OTP being generated is for the
+        # same number that belongs to this user.
+
+        # A session should already exist since this should be called after the first
+        # factor is completed
+        _session = await get_session(api_options.request)
+        assert _session is not None
+
+        # highlight-next-line
+        payload_phone_number = _session.get_access_token_payload().get("phone_number")
+
+        if payload_phone_number is not None:
+            # this means we found a phone number associated to this user
+            # we will check if the input phone number is the same as this one.
+            if phone_number is not None and phone_number != payload_phone_number:
```

Comment:
Incorrect condition as explained in previous code snippet

Reply:
done.